source/agora/common/Deserializer.d

```diff
@@ -143,95 +144,103 @@ public T deserializeFull (T) (scope ubyte[] data) @safe
 }
 
 /// Ditto
-public T deserializeFull (T) (scope DeserializeDg dg) @safe
+public void deserializePart (T) (ref T record, scope DeserializeDg dg) @safe
```

Comment:
I was slightly confused at the diff at first, but then I realized that you've changed the order of the functions too.

Reply:
I kept `deserializePart` as a shell for `deserializeFull`, since most `deserialize` functions defined in `struct` use it and I didn't want to diff to be too big.